EricVook-Albuquerque.md

```diff
@@ -0,0 +1,13 @@
+## Albuquerque, NM
+
+### Great places to eat
+
+
+### Fun things to do
+
+Visit Sandia Peak
```

Comment:
> Looks like you made a mistake in line 3.

I think it ok, but I will make it a sub-heading which will probably look better.  Good comment, @buchhei .